### PR TITLE
Add resource for kid list.

### PR DESCRIPTION
**Pull request by ArturFriesen**
## Description
Adds resource for kid list request.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12575

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/HTTPClient/Resources/DCCRevocation/KIDList/KIDListResource.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		}
+	}
+	
+	// MARK: - Private
+	
+	private func unexpectedServerError(
+		_ statusCode: Int
+	) -> KIDListResourceError? {
+		switch statusCode {
+		case (400...409):
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/HTTPClient/Resources/DCCRevocation/KIDList/KIDListResource.swift:56]:
```suggestion
		case (400...499):
```

File: src/xcode/ENA/ENA/Source/HTTPClient/Resources/DCCRevocation/KIDList/KIDListResource.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+			return .DCC_RL_KID_LIST_CLIENT_ERROR
+		case (509...509):
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/HTTPClient/Resources/DCCRevocation/KIDList/KIDListResource.swift:58]:
```suggestion
		case (500...599):
```

**Review by pascal-brause — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/HTTPClient/Resources/DCCRevocation/KIDList/KIDListResource.swift (hunk 2 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/HTTPClient/Resources/DCCRevocation/KIDList/KIDListResource.swift]: Changed

File: src/xcode/ENA/ENA/Source/HTTPClient/Resources/DCCRevocation/KIDList/KIDListResource.swift (hunk 1 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/HTTPClient/Resources/DCCRevocation/KIDList/KIDListResource.swift]: changed

**Review by pascal-brause — APPROVED** [review]

**Review by pascal-brause — APPROVED** [review]
It's really easy work, but could you add the test for the unique identifier for the new resource?
And what about the unit tests itself?

**Review by KaiTeuber — APPROVED** [review]

**Pull request merged**